server.ts

```diff
@@ -3,16 +3,14 @@ import configureApp from './app';
 
 const app = configureApp();
 const server: http.Server = new http.Server(app);
-const PORT = process.env.PORT || 9999;
+const PORT = process.env.PORT ?? 9999;
 
 server.listen(PORT);
 
 server.on('error', (e: Error) => {
-  console.log('Error starting server' + JSON.stringify(e));
+  console.log(`Error starting server${JSON.stringify(e)}`);
```

Comment:
Should there be a space here?
```suggestion
  console.log(`Error starting server ${JSON.stringify(e)}`);
```

Reply:
Probably? But that's not how it was written. Seems like a small enough change that I can add it no problem.